src/components/FieldNote/FieldNote.module.css

```diff
@@ -11,31 +11,37 @@
   display: inline-block;
   @mixin eds-theme-typography-body-text-xs;
   margin-top: var(--eds-size-half);
-  color: var(--eds-theme-color-neutral-md-foreground);
+  color: var(--eds-theme-color-text-neutral-subtle);
 
   /**
      * Field note inside a field that has an error 
      */
-  .eds-is-error & {
-    color: var(--eds-theme-color-utility-error-foreground);
+  &.eds-is-error {
+    color: var(--eds-theme-color-text-utility-error);
   }
 }
 
 /**
   * Fieldnote icon
   */
-.field-note__icon {
+.field-note__icon.field-note__icon {
+  height: var(--eds-size-2);
+  width: var(--eds-size-2);
+  position: relative;
+  bottom: 1px;
+
   .eds-is-error & {
     display: inline-block;
-    margin-right: var(--eds-size-1);
     position: relative;
     top: 3px;
+    margin-right: var(--eds-size-1);
+    fill: currentColor; /* 1 */
```

Comment:
This is the default behavior for icons now, so extra `fill` styling not needed. See prop destructuring in `Icon.tsx`